assets/tests/helpers/sentryInit.test.ts

```diff
@@ -0,0 +1,9 @@
+import sentryInit from "../../src/helpers/sentryInit"
+
+const opts = {
+  environment: "test_env",
+}
+
+test("Sentry Init runs", () => {
+  expect(sentryInit(opts)).toHaveProperty("environment", "test_env")
```

Comment:
So this is testing that the sentryInit function returns the options passed in, but what we'd really like to test is that Sentry.init gets called with the options. We can do this with a combination of imports and use of jest.mock like so (it's sort of complicated so I'll just paste the whole result rather than trying to walk through the changes individually):

```
import sentryInit from "../../src/helpers/sentryInit"
import * as Sentry from "@sentry/react"

jest.mock("@sentry/react", () => ({
  __esModule: true,
  init: jest.fn(),
}))

const opts = {
  environment: "test_env",
}

test("Sentry Init runs", () => {
  sentryInit(opts)
  expect(Sentry.init).toHaveBeenCalledWith(opts)
})
```

With that change made we can also remove the return and make sentryInit return void since the return value isn't actually used anywhere.

Also, if you're here anyway, it might be nice to write a test for Sentry.setUser part.

Reply:
updated thanks! also added a test for Sentry.setUser and moved the window.username into an input to the function for easier testing